src/get-queries-for-element.js

```diff
@@ -7,14 +7,19 @@ import * as defaultQueries from './queries'
 /**
  * @param {HTMLElement} element container
  * @param {FuncMap} queries object of functions
+ * @param {Object} initialValue for reducer
```

Comment:
Interesting idea. This probably isn't how I would have done it, but I'm actually pretty cool with it.